source/development_process/index.rst

```diff
@@ -85,12 +85,25 @@ Determine Requirements
 
 Setting Up the Project
 ----------------------
-- Create the git repo
-- Create the issue tracker
+
+**Create the git repo**: Unless you have special permission in the `OSL
+organization github <https://github.com/osuosl>`_, you will probably
+need to ask a full-time employee to make the repository for you.  We
```

Comment:
I read the little instructions. I would argue this makes the document feel very impersonal and scientific in an undesirable way -- we aren't business majors here -- but if you insist.